x-pack/plugins/index_lifecycle_management/public/store/selectors/policies.js

```diff
@@ -212,17 +213,17 @@ export const policyFromES = (policy) => {
   };
 };
 
-export const phaseToES = (state, phase) => {
-  const esPhase = {};
+export const phaseToES = (state, phase, originalEsPhase) => {
```

Comment:
Nit: `state` is not used and could be removed from the arguments. (and from the call in `lifecycle.js`)